src/internal/tester/tconfig/protected_resources.go

```diff
@@ -278,9 +278,10 @@ func SecondaryM365TeamID(t *testing.T) string {
 // by either the env var CORSO_M365_TEST_GROUP_ID, the corso_test.toml config
 // file or the default value (in that order of priority).  The default is a
 // last-attempt fallback that will only work on alcion's testing org.
+// NOTE: This Group will not have a Team associated with it.
 func M365GroupID(t *testing.T) string {
```

Comment:
This is now only used for a single test, the new one added in this PR. All other instances have been changd to `M365TeamID`.